src/tests/unit/tests/electron/views/screenshot/screenshot-view.test.tsx

```diff
@@ -5,10 +5,22 @@ import { shallow } from 'enzyme';
 import * as React from 'react';
 
 describe('ScreenshotView', () => {
-    it('renders', () => {
-        const props: ScreenshotViewProps = {};
-        const wrapper = shallow(<ScreenshotView {...props} />);
+    describe('renders', () => {
+        it('when passed a value for screenshotData', () => {
+            const props: ScreenshotViewProps = {
+                screenshotData: { base64PngData: 'string' },
```

Comment:
instead of `string` which is a type in typescript, would it be better to have something in the lines of `"test-base-64-png-data"`??

Reply:
changing!